setup.py

```diff
@@ -65,10 +65,10 @@
     'sphinxcontrib-programoutput',
     ]
 
-testing_extras = tests_require + [
-    'nose',
-    'coverage',
-    'virtualenv', # for scaffolding tests
+tests_require = [
+    'WebTest >= 1.3.1',  # py3 compat
+    'pytest',  # includes virtualenv
+    'pytest-cov',
```

Comment:
Do note change these deps for now... keep the list containing what it contained before.